extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -3332,7 +3349,7 @@ static public function DBtoUrl( $dbname, $master = false ) {
 	 * @param $ids array List of wiki ids
 	 * @param $what int Flags specifying what data to prefetch
 	 */
-	static public function prefetchWikisById( $ids, $what = self::PREFETCH_DEFAULT ) {
+	static public function prefetchWikisById( $ids, $what = static::PREFETCH_DEFAULT ) {
 		global $wgMemc;
 		if ( !is_array( $ids ) ) $ids = [ $ids ];
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add curly braces around the nested statement(s). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS121)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) 2 statements were found on this line. Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS122)